source/geometry/IntTransform2D.ooc

```diff
@@ -57,6 +58,7 @@ IntTransform2D: cover {
 	}
 	determinant ::= this a * this e * this i + this d * this h * this c + this g * this b * this f - this g * this e * this c - this d * this b * this i - this a * this h * this f
 	translation ::= IntVector2D new(this g, this h)
+	diagonal ::= IntVector3D new(this a, this e, this i)
```

Comment:
I don't know if this should be here, the class is 2D and returns something 3D. @marcusnaslund